### ISSUE TITLE
Padding not applied to ButtonView via Style

### ISSUE THREAD

**Issue by pikausp** [enisn/UraniumUI#494]
When I declare a style that targets `ButtonView` the `Padding` property is ignored. Setting `Padding` directly on the control works as expected. Other properties, such as `Margin` seem to work just fine. Am I missing Something?

```
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:material="http://schemas.enisn-projects.io/dotnet/maui/uraniumui/material"
             xmlns:t="clr-namespace:UraniumUI.Theming;assembly=UraniumUI"
             x:Class="UraniumStyleIsolation.MainPage">
    <ContentPage.Resources>
        <Style TargetType="material:ButtonView">
            <Setter Property="Padding" Value="0"/>
        </Style>

        <Style TargetType="material:ButtonView" Class="Margin">
            <Setter Property="Margin" Value="50"/>
        </Style>
    </ContentPage.Resources>

    <ScrollView>
        <VerticalStackLayout>
            <material:ButtonView Margin="20"
                                 BackgroundColor="Black">
                <Label Text="Test"
                       BackgroundColor="Gray"/>
            </material:ButtonView>

            <material:ButtonView Margin="20"
                                 Padding="0"
                                 BackgroundColor="Blue">
                <Label Text="Test"
                       BackgroundColor="Red"/>
            </material:ButtonView>

            <material:ButtonView Padding="0"
                                 BackgroundColor="Green"
                                 StyleClass="Margin">
                <Label Text="Test"
                       BackgroundColor="Yellow"/>
            </material:ButtonView>


        </VerticalStackLayout>
    </ScrollView>

</ContentPage>
``` 

![image](https://github.com/enisn/UraniumUI/assets/6717146/8249447a-143e-4b57-9b33-ff9971b306d6)

**Comment by enisn** [comment]
I'll consider converting the default styling method to CSS files:
https://learn.microsoft.com/en-us/dotnet/maui/user-interface/styles/css?view=net-maui-8.0

So, entire resources of specific elements won't be overridden

**Comment by pikausp** [comment]
If I understand it correctly this is not a bug with an easy fix, but rather it requires a fundamental update to the library. Does that mean I should set `Padding` on each element separately instead of via a style for the time being?

**Comment by enisn** [comment]
Sorry, I couldn't explain well. I've done required changed in the existing code and released with `v2.7.1`. 

But, in the long term I'll convert the usage to CSS files, so cascading styling can be done better

I've updated documentation for **StatefulContentView** and **ButtonView**

https://enisn-projects.io/docs/en/uranium/latest/infrastructure/StatefulContentView#customizations

https://enisn-projects.io/docs/en/uranium/latest/themes/material/ButtonView#customizations

If you use `BaseResourceKey` in your style, you'll inherit the existing style _(Probably you inherit without writing it. I haven't tested. But defining it is better)_

**Comment by pikausp** [comment]
Thanks a lot! Just noticed it was hard coded in the constructor, so the behavior makes sense now.
